lighthouse-cli/cli-flags.js

```diff
@@ -7,18 +7,23 @@
 
 /* eslint-disable max-len */
 
-const yargs = require('yargs');
-const fs = require('fs');
-const {isObjectOfUnknownValues} = require('../lighthouse-core/lib/type-verifiers.js');
+import * as fs from 'fs';
+
+import yargs from 'yargs';
+import * as yargsHelpers from 'yargs/helpers';
+
+import {isObjectOfUnknownValues} from '../lighthouse-core/lib/type-verifiers.js';
 
 /**
  * @param {string=} manualArgv
  * @param {{noExitOnFailure?: boolean}=} options
  * @return {LH.CliFlags}
  */
 function getFlags(manualArgv, options = {}) {
-  // @ts-expect-error - undocumented, but yargs() supports parsing a single `string`.
-  const y = manualArgv ? yargs(manualArgv) : yargs;
+  const y = manualArgv ?
+    // @ts-expect-error - undocumented, but yargs() supports parsing a single `string`.
+    yargs(manualArgv) :
+    yargs(yargsHelpers.hideBin(process.argv));
```

Comment:
Why the change? Seems way worse than having `yargs` handle it :)

Reply:
oh. at one point I may have tried updating yargs and found this to be necessary. I can probably revert.

Reply:
hmm... not sure what changed, as there is no yargs upgrade, but the old code results in this error:


```
TypeError: y.help is not a function
    at getFlags (file:///Users/cjamcl/src/lighthouse/lighthouse-cli/cli-flags.js:28:18)
    at begin (file:///Users/cjamcl/src/lighthouse/lighthouse-cli/bin.js:53:20)
    at file:///Users/cjamcl/src/lighthouse/lighthouse-cli/index.js:11:1
    at ModuleJob.run (internal/modules/esm/module_job.js:152:23)
    at async Loader.import (internal/modules/esm/loader.js:177:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)
```

Reply:
> hmm... not sure what changed, as there is no yargs upgrade, but the old code results in this error:

must be some change to the esmodules export. Here's a similar case: https://github.com/yargs/yargs/issues/1774

Reply:
shurg. this is exactly what their demo code does https://github.com/yargs/yargs#esm

tweaking the import to be `*` creates type errors.